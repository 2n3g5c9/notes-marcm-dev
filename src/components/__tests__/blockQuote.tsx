import React from 'react'
import renderer from 'react-test-renderer'

import { BlockQuote } from '@components'

jest.mock('gatsby-plugin-mdx', () => {
  return {
    // eslint-disable-next-line react/display-name
    MDXRenderer: ({ children }: { children: React.ReactNode }) => {
      return <div>{children}</div>
    },
  }
})

describe('BlockQuote', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<BlockQuote>Test</BlockQuote>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
