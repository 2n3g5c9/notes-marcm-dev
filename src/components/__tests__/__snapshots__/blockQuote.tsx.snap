// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockQuote renders correctly 1`] = `
<div
  className="bg-secondary border-primary border-l-2 sm:border-l-4 font-mono inline-block my-2 px-4 py-3 relative text-secondary text-xs sm:text-sm sm:w-auto w-full"
>
  <div
    className="overflow-x-auto whitespace-pre"
  />
  <button
    className="absolute bg-primary font-sans m-2 px-2 py-1 right-0 rounded select-none top-0"
    onClick={[Function]}
  >
    <div
      style={
        Object {
          "position": "relative",
        }
      }
    />
    Copy
  </button>
</div>
`;
